tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/hotupdate/recreate/RecreateUpdateStrategyTest.java

```diff
@@ -80,12 +79,6 @@ public void checkRecreateUpdateStrategy() throws Exception {
 
     cheTestSystemClient.stop();
 
-    // open the Dashboard and make sure that workspace is not available after suspending system
```

Comment:
This verification looks important, as for me. Could you, please, explain why do we need to remove this steps?

Reply:
Now the behaviour is changed. After stop `che-server` by API the dashboard is not open more and there is present only `che loader` page. 
I had conversation with @olexii4  (who made changes in the behaviour) and actually both variants are not ideal and require discussion.
But it is not a bug and we should continue checking this functionality as is.

Reply:
Got it. Thanks for explanation!